File: src/js/fibo.js
function fibo(n) {
    
    if(n==0)
        return 0;
    
    let f0 = 0;
    let f1 = 1;
    
    
    for(let i = 1; i<n; i++)
    {
        let tmp = f0;
        f0 = f1;
        f1 += tmp;
    }
        
    return f1;
}

File: src/rust/fibo.rs
fn fibo(n: u64) -> u64
{
    let mut f0: u64 = 0;
    let mut f1: u64 = 1;

    if n<=1
    {
        return n;
    }

    for _ in (1..n)
    {
        let old_f0 = f0;
        f0 = f1;
        f1 += old_f0;
    }

    return f1;
}